www/src/pages/packages.js

```diff
@@ -0,0 +1,55 @@
+import React, {Component} from "react"
+import Container from "../components/container"
+import logo from "../gatsby-negative.svg"
+import { rhythm } from "../utils/typography"
+
+class Plugins extends Component {
+  render(){
+    return(
+      <Container>
+        <div
+          css={{
+            display: `flex`,
+            flexDirection: `column`,
+            alignItems: `center`,
+            justifyContent: `center`,
+            height: `calc(100vh - 300px)`,
+          }}
+          >
+          <img src={logo}
+            css={{
+              display: `inline-block`,
+              height: rhythm(5.2),
+              width: rhythm(5.2),
+              marginLeft: `auto`,
+              marginRight: `auto`,
+            }}
+            alt=""
+
+          />
+          <h1
+            css={{
+              fontSize: rhythm(1),
+              marginTop: rhythm(1/4),
+              marginLeft: rhythm(1),
+              marginRight: rhythm(1),
+              textAlign: `center`,
+            }}
+
+            >Welcome to the Gatsby Plugin and Starter Library!</h1>
```

Comment:
Let's change "plugin and starter" to just "package" since we're not going to search for starters here